goldenmaster/examples/qml/main.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.2
+import QtQuick.Controls 2.15
+import testbed2 1.0
+import tb_enum 1.0
+import tb_same1 1.0
+import tb_same2 1.0
+import tb_simple 1.0
+import testbed1 1.0
+
+ApplicationWindow {
+    id: appWindow
+    visible: true
+    width: 300
+    height: 300
+    ColumnLayout {
+        spacing: 10
+        id: mainLayout
+        anchors.fill: parent
+    Testbed2ManyParamInterface { id: qmlTestbed2ManyParamInterface }

```

Comment:
the interfaces should be on the root level at the end. So the visual aspects are shown first.
```qml
ApplicationWindow {
  ColumnLayout { ... }
  // interfaces ...
}
```